### PR TITLE
Disable native maven based tests

### PR DESCRIPTION
**Pull request by timja**
### Proposed changelog entries

* Internal: Change tests to retrieve maven over https due to upstream requiring https only
* ...

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [na] JIRA issue is well described
- [x] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@jenkinsci/core-pr-reviewers 

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a JIRA issue should exist and be labeled as `lts-candidate`

### CODE REVIEW

**Review by fcojfernandez — APPROVED** [review]

**Review by oleg-nenashev — APPROVED** [review]
Thanks for the quick fix! I will merge it once the build passes

**Review by MRamonLeon — APPROVED** [review]

**Review by varyvol — APPROVED** [review]

**Comment by timja** [comment]
I think more changes are needed unfortunately 😢 
https://stackoverflow.com/a/59764670/4951015

Only recent maven uses https by default

**Comment by timja** [comment]
The specific test that is failing passes locally, might be to do with the maven version running on CI

**Comment by timja** [comment]
Managed to reproduce locally,
I had maven-clean-plugin:2.2 cached in my local repository, and then after removing it, it was using my jenkins-ci mirror I have configured, after disabling that I can reproduce it

**Comment by timja** [comment]
Might need to get someone else to re-submit this, cc @jenkinsci/core 
```
19:42:33  Obtained Jenkinsfile from d02c7563588aa534629e05b873946680953dbfdc
[2020-01-17T19:42:34.292Z] ‘Jenkinsfile’ has been modified in an untrusted revision
```

**Comment by oleg-nenashev** [comment]
on it

**Comment by oleg-nenashev** [comment]
If #4437 does not play out, will just mark tests as ignored for now

File: Jenkinsfile
Comment (MarkEWaite) [inline Jenkinsfile]: Could we avoid the whitespace change in this line?

**Review by MarkEWaite — APPROVED** [review]
Minor comment about restoring a line with trailing whitespace. Does not block the merge

File: Jenkinsfile
Comment (timja) [reply Jenkinsfile]: it's been reverted

**Review by oleg-nenashev — APPROVED** [review]
Would be nice to have a follow-up task in Jira. Will merge once the build passes

**Comment by oleg-nenashev** [comment]
One test is still failing. I might find some time to fix/ignore it tonight, but no committment.

**Comment by timja** [comment]
> One test is still failing. I might find some time to fix/ignore it tonight, but no committment.

https://github.com/jenkinsci/jenkins/pull/4435/commits/6a34d5fb9d91acdee9e2f3bc92c994dfd8ff4246

`@Ignore doesn't work with junit.framework.TestCase` based tests

**Comment by timja** [comment]
Build is green after the last kick

Follow-up jira issue:
https://issues.jenkins-ci.org/browse/JENKINS-60813

**Comment by oleg-nenashev** [comment]
Thanks @timja!

**Pull request merged**
